plsync/planetlab/model.py

```diff
@@ -589,6 +608,29 @@ def __init__(self, **kwargs):
 
         super(Slice, self).__init__(**kwargs)
 
+    def dnsname(self):
+        """Returns the slice name in DNS form, e.g. group_name to name.group"""
+        fields = self['name'].split('_')
+        # When a name has multiple '_', rejoin all parts after the group name.
```

Comment:
We don't have any slices with more than a single underscore that I know of. But, it's allowed by PL. To some degree, there is an expectation that we remain interoperable with PL. In general, I think we want zero underscores in our experiment names. The mismatch between PL slice names and DNS names has been a wart for a while.

Thinking this through, i noticed the code does not support the "zero underscore" name. So, I updated the code to support these three forms: "name", "group_name", or "group_weird_name".

How does it look?